recipes/r-recetox-xmsannotator/meta.yaml

```diff
@@ -8,42 +8,62 @@ package:
 
 source:
   url: "{{ github }}/archive/refs/tags/v{{ version }}.tar.gz"
-  sha256: b59eaae4e76c917be486b3b2568b010350e246ec510fb284c4697e6f71a17299
+  sha256: 362abcd78bff8ca8149efa5e66d43611e5e8bb23443db84c33fa026af0aa6eaa
 
 build:
-  number: 1
+  number: 0
   noarch: generic
   rpaths:
     - lib/
     - lib/R/lib/
-  script: R CMD INSTALL --build xmsannotator
+  script: R CMD INSTALL --build .
 
+channels:
+  - conda-forge
+  - bioconda
 
 requirements:
   host:
-    - r-base
-    - r-dplyr >=1
+    - r-base =4.1.1
```

Comment:
I would not pin it that strict if possible.

Reply:
When using the environment script to create the environment, pinning the R version massively reduces the time needed to resolve. Not sure how this maps to the host section of a recipe.